messages/notify.go

```diff
@@ -116,6 +131,8 @@ func (n *Notify) DecodeFromBytes(b []byte) error {
 
 // SetLength sets the length in Length field.
 func (n *Notify) SetLength() {
+	n.Status.SetLength()
```

Comment:
Please add nil check, as it's not always guaranteed to have `Status' param. I want it to be strict enough not to panic, as even if the it's validated in SerializeTo(), it is not guaranteed for `Notify` to have `Status` every time this exported method is called.

Reply:
Fixed at 94f67ffa234703d439ee0b406d08b241703b6fd7

Reply:
Confirmed. Thanks.